Apple-TV/Playback/VLCFullscreenMovieTVViewController.m

```diff
@@ -831,28 +832,25 @@ - (void)mediaPlayerStateChanged:(VLCMediaPlayerState)currentState
     }
 }
 
-- (void)displayMetadataForPlaybackController:(VLCPlaybackController *)controller
-                                       title:(NSString *)title
-                                     artwork:(UIImage *)artwork
-                                      artist:(NSString *)artist
-                                       album:(NSString *)album
-                                   audioOnly:(BOOL)audioOnly
+- (void)displayMetadataForPlaybackController:(VLCPlaybackController *)controller metadata:(VLCMetaData *)metadata
 {
+    NSLog(@"%@",metadata);
```

Comment:
Consider using APLog here to not leak the info in production.

Reply:
ah that can even go away! totally missed deleting that